### PR TITLE
[TE] onboarding - classic task execution

### PR DESCRIPTION
**Pull request by apucher**
The existing self-serve implementation uses a reverse proxy to a backend node as a backstop solution. This PR removes the reverse proxy and consolidates replay execution state in the database 

* move task creation for replay into webapp

* wrap onboarding framework execution in classic TE task

* eliminate reverse proxy for self-serve

### CODE REVIEW

**Comment by kishoreg** [comment]
Awesome job!. Thanks for cleaning this up.

**Comment by apucher** [comment]
this is hardly a cleanup. It's just another hack on top to get rid of the reverse proxy. @jihaozh and I will look into a more robust solution

**Comment by cyenjung** [comment]
LGTM. Please address @cyy0714's concern before pushing the code. Thanks for improving this piece of code.

File: thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/AnomalyFunctionBean.java [hunk 1]
```diff
@@ -296,37 +299,50 @@ public void setAnomalyMergeConfig(AnomalyMergeConfig anomalyMergeConfig) {
...
 
+  public void setWindowSize(Integer windowSize) {
+    this.windowSize = windowSize;
+  }
+
+  public void setWindowDelay(Integer windowDelay) {
+    this.windowDelay = windowDelay;
+  }
+
+  public DetectionOnboardJobStatus getOnboardJobStatus() {
```
[the remark this answers is not in the document]
Comment (apucher) [reply thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/datalayer/pojo/AnomalyFunctionBean.java]: We need a reference for anomaly function to task status. One way or another we'll have to modify the AnomalyFunctionBean to store a reference to a task dto or the status directly. If you keep it separate there's just another thing you need to keep in sync (e.g. what happens if one of them is "cleaned up"?)

**Pull request merged**